File: src/main/java/ru/translater/common/model/Meta.java
package ru.translater.common.model;

import lombok.Data;
import org.jsoup.internal.StringUtil;

@Data
public class Meta {
    private String name;
    private String property;
    private String content;

    public boolean isNameExists() {
        return StringUtil.isBlank(property) && !StringUtil.isBlank(name);
    }
}

File: src/main/kotlin/com/translater/common/model/Meta.kt
package com.translater.common.model


data class Meta(
    var name: String? = null,
    var property: String? = null,
    var content: String? = null
) {
    val isNameExists: Boolean by lazy {
        property?.isBlank() ?: false && name?.isNotBlank() ?: false
    }
}